doc/changelog/01-kernel/10439-uniform-opaque-seff-handling.md

```diff
@@ -0,0 +1,4 @@
+- Internal definitions generated by abstract-like tactics are now inlined
+  inside universe polymorphic definitions, similarly to what happens for their
```

Comment:
Doesn't inlining only happen for Qed'd definitions (in the monomorphic case)?  Can you clarify the behavior of abstract in polymorphic definitions wrt Qed vs Defined?

Reply:
Yes, this is missing from the description, it only happens for Qed-terminated definitions.